### PR TITLE
Test cases when attributes are set in all server.env, bootstrap.properties and server.xml

### PR DESCRIPTION
**Pull request by leemelim**
New test cases in logging FATs to test the scenario, when logging configurations are specified in both bootstrap.properties and server.xml.

server.xml takes precedence over bootstrap.properties once the server is started.

### CODE REVIEW

**Comment by leemelim** [comment]
#build

**Comment by leemelim** [comment]
#build

**Comment by leemelim** [comment]
#build

**Comment by leemelim** [comment]
#build

File: dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/ConsoleFormatTest.java [hunk 1]
```diff
@@ -420,9 +475,10 @@ private void writeProperties(Properties props, FileOutputStream output) throws E
         }
     }
 
-    private static void setServerConfiguration(String consoleFormat, boolean useTraceSpec, boolean useIsoDateFormat, RemoteFile consoleLogFile) throws Exception {
+    private static void setServerConfiguration(LibertyServer Server, String consoleFormat, boolean useTraceSpec, boolean useIsoDateFormat,
```
Comment (pgunapal) [inline dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/ConsoleFormatTest.java:478]: The variable name "Server" should be in lowercase, to meet java standards. Maybe, rename it to something else, so it doesn't conflict with "server"?

File: dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/ConsoleFormatTest.java [hunk 2]
```diff
@@ -444,10 +500,10 @@ private static boolean isStringinSimpleFormat(String text) {
         return Pattern.compile(SIMPLE_FORMAT_REGEX_PATTERN).matcher(text).find();
     }
 
-    private void setInBootstrapPropertiesFile(RemoteFile bootstrapFile, String key, String value) throws Exception {
+    private void setInBootstrapPropertiesFile(LibertyServer Server, RemoteFile bootstrapFile, String key, String value) throws Exception {
```
Comment (pgunapal) [inline dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/ConsoleFormatTest.java:503]: The variable name "Server" should be in lowercase, to meet java standards. Maybe, rename it to something else, so it doesn't conflict with "server"?

**Comment by leemelim** [comment]
#build

**Comment by leemelim** [comment]
fixes #11358 
#run-libby-bot

**Pull request merged**